app-ios/Sources/SessionFeature/TagView.swift

```diff
@@ -0,0 +1,19 @@
+import SwiftUI
+
+struct Tag: Equatable, Hashable {
+    let text: String
+    let color: Color
+}
+
+extension Tag {
+    var view: some View {
+        return Text(self.text)
+            .font(Font.system(size: 14, weight: .regular, design: .default))
+            .padding(.horizontal)
+            .padding(.vertical, 4)
+            .background(
+                Capsule()
+                    .fill(self.color)
+            )
+    }
+}
```

Comment:
How about use `CapsuleText` that is introduced in https://github.com/DroidKaigi/conference-app-2022/pull/402 🙏🏼 
But that view is defined in `TimetableFeature`, you can add that file copy in `SessionFeature`. If so, I'll define `CommonComponent` target and move those with another PR.

Reply:
Sounds fine! For the meantime I'll copy it in Session 👍 CommonComponents in a future PR sounds good.

Reply:
Added a  copy of `CapsuleText` in SessionFeature 👍